readme.md

```diff
@@ -4,7 +4,7 @@
 
 # Microsoft Azure Event Hubs Client for .NET
 
-> Please note, newer packages [Azure.Messaging.EventHubs](https://www.nuget.org/packages/Azure.Messaging.EventHubs) and [Azure.Messaging.EventHubs.Processor](https://www.nuget.org/packages/Azure.Messaging.EventHubs.Processor) for [Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) are available as of February 2020. While the packages `Microsoft.Azure.EventHubs` and `Microsoft.Azure.EventHubs.Processor` will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the [migration guide](https://aka.ms/azsdk/net/migrate/eh) for more details.
+> Please note, newer packages [Azure.Messaging.EventHubs](https://www.nuget.org/packages/Azure.Messaging.EventHubs) and [Azure.Messaging.EventHubs.Processor](https://www.nuget.org/packages/Azure.Messaging.EventHubs.Processor) for [Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) are available as of February 2020. While the packages `Microsoft.Azure.EventHubs` and `Microsoft.Azure.EventHubs.Processor` will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the [migration guide](https://aka.ms/azsdk/net/migrate/eh) for more details. `Microsoft.Azure.EventHubs.ServiceFabricProcessor` is also deprecated and won't be receiving any changes. It never advanced beyond preview and the [new Event Hubs clients](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/eventhub/Azure.Messaging.EventHubs) are pursuing other routes for Service Fabric integration. 
```

Comment:
I would recommend moving the part on ServiceFabricProcessor to a separate note of its own. Otherwise, it looks to be buried inside the EH notes

```suggestion
> Please note, newer packages [Azure.Messaging.EventHubs](https://www.nuget.org/packages/Azure.Messaging.EventHubs) and [Azure.Messaging.EventHubs.Processor](https://www.nuget.org/packages/Azure.Messaging.EventHubs.Processor) for [Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) are available as of February 2020. While the packages `Microsoft.Azure.EventHubs` and `Microsoft.Azure.EventHubs.Processor` will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the [migration guide](https://aka.ms/azsdk/net/migrate/eh) for more details. 

> `Microsoft.Azure.EventHubs.ServiceFabricProcessor` is also deprecated and won't be receiving any changes. It never advanced beyond preview and the [new Event Hubs clients](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/eventhub/Azure.Messaging.EventHubs) are pursuing other routes for Service Fabric integration. 
```

Reply:
> pursuing other routes for Service Fabric integration

There is no ongoing work on this. So, I would refrain from mentioning it. If we get feedback from folks asking for alternatives, we can then use that to decide what to do next

Reply:
Thanks, @ramya-rao-a . Updated the PR with suggested changes.

Reply:
@ramya-rao-a I actually have two upcoming projects where I was considering using Service Fabric to provide a stable platform for Event Hub message processing. My only (straightforward) alternatives at the moment, as far as I'm aware, are Functions (which have less predictable/stable tail latencies and are not optimal for larger throughput volumes) and Stream Analytics (which has limited expressive/routing power). For reference, my scenario involves transforming events and then pushing them to Digital Twins.